support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -101,6 +101,7 @@ type GiftRecipientType = {|
   lastName: string,
   email?: Option<string>,
   // message: Option<string>,
+  // deliveryDate: Option<string>,
```

Comment:
this is a placeholder to be done when this PR is merged: https://github.com/guardian/support-frontend/pull/2702#discussion_r494923958